airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AdvancedAcceptanceTests.java

```diff
@@ -319,61 +318,31 @@ public void testDowntimeDuringSync() throws Exception {
     final DestinationSyncMode destinationSyncMode = DestinationSyncMode.OVERWRITE;
     catalog.getStreams().forEach(s -> s.getConfig().syncMode(syncMode).destinationSyncMode(destinationSyncMode));
 
-    for (final var input : List.of("KILL_BOTH_NON_SYNC_SLIGHTLY_FIRST", "KILL_ONLY_SYNC", "KILL_ONLY_NON_SYNC")) {
-      LOGGER.info("Checking " + input);
-
-      final UUID connectionId =
-          testHarness.createConnection(connectionName, sourceId, destinationId, List.of(), catalog, null).getConnectionId();
+    LOGGER.info("Creating connection...");
+    final UUID connectionId =
+        testHarness.createConnection(connectionName, sourceId, destinationId, List.of(), catalog, null).getConnectionId();
 
-      JobInfoRead connectionSyncRead = null;
+    LOGGER.info("Run manual sync...");
+    JobInfoRead connectionSyncRead = apiClient.getConnectionApi().syncConnection(new ConnectionIdRequestBody().connectionId(connectionId));
 
-      while (connectionSyncRead == null) {
+    LOGGER.info("Waiting for job to run...");
+    waitWhileJobHasStatus(apiClient.getJobsApi(), connectionSyncRead.getJob(), Set.of(JobStatus.PENDING));
 
-        try {
-          connectionSyncRead = apiClient.getConnectionApi().syncConnection(new ConnectionIdRequestBody().connectionId(connectionId));
-        } catch (final Exception e) {
-          LOGGER.error("retrying after error", e);
-        }
-      }
+    LOGGER.info("Scaling down worker...");
+    kubernetesClient.apps().deployments().inNamespace("default").withName("airbyte-worker").scale(0, true);
 
-      Thread.sleep(10000);
-
-      switch (input) {
-        case "KILL_BOTH_NON_SYNC_SLIGHTLY_FIRST" -> {
-          LOGGER.info("Scaling down both workers at roughly the same time...");
-          kubernetesClient.apps().deployments().inNamespace("default").withName("airbyte-worker").scale(0);
-          kubernetesClient.apps().deployments().inNamespace("default").withName("airbyte-sync-worker").scale(0, true);
```

Comment:
Removed these inputs and switch statement, because we don't have any deployments called `airbyte-sync-worker`. I think this was leftover from some other changes or something